include/zephyr/net/wifi_mgmt.h

```diff
@@ -245,7 +245,7 @@ struct wifi_scan_params {
 	uint16_t dwell_time_passive;
 	/** Array of SSID strings to scan.
 	 */
-	char ssids[WIFI_MGMT_SCAN_SSID_FILT_MAX][WIFI_SSID_MAX_LEN + 1];
+	const char *ssids[WIFI_MGMT_SCAN_SSID_FILT_MAX];
```

Comment:
So the driver either needs to copy the ssid values or the caller needs to keep the pointer values in some non stack location. Should we emphasize this in the documentation two lines above? For example a note saying that the caller should keep the pointer values valid for the duration of the scan or something like that?

Reply:
This is already the case with the current setup as there is no guarantee in the API that the `struct wifi_scan_params` will remain valid after triggering `NET_REQUEST_WIFI_SCAN`.

Reply:
Ok, then I am suggesting that it might make sense to add some comment about this here. Could you add one?